### PR TITLE
DashboardScene: Map timezone and fiscal year to scene model and back

### PR DESCRIPTION
**Pull request by torkelo**
* [x] Maps timezone to runtime scene model and back
* [x] Maps fiscal year to runtime scene model and back
* [x] Map weekStart (needs support in scene lib)

Scene PR for weekStart and fiscalYearMonth fix https://github.com/grafana/scenes/pull/375

### CODE REVIEW

**Review by axelavargas — COMMENTED** [review]
Hey @torkelo  👋🏾, I gave it a try locally and noticed that the timezone doesn't seem to be preserved when I toggle the 'scenes' button. Is this expected behavior, or should we see the core changes in scene rendering even without saving?

https://github.com/grafana/grafana/assets/239999/5ef07c70-2cb5-4374-abc8-dfcbb06b5ae7

**Comment by torkelo** [comment]
@axelavargas I did not see you save your change

**Comment by torkelo** [comment]
@axelavargas this PR is only touching the timezone dashboard settings option 
image

File: public/app/features/dashboard-scene/serialization/transformSaveModelToScene.ts [hunk 1]
```diff
@@ -208,7 +208,13 @@ export function createDashboardSceneFromDashboardModel(oldModel: DashboardModel)
       isLazy: true,
       children: createSceneObjectsForPanels(oldModel.panels),
     }),
-    $timeRange: new SceneTimeRange(oldModel.time),
+    $timeRange: new SceneTimeRange({
+      from: oldModel.time.from,
+      to: oldModel.time.to,
+      fiscalYearStartMonth: oldModel.fiscalYearStartMonth,
```
Comment (axelavargas) [inline public/app/features/dashboard-scene/serialization/transformSaveModelToScene.ts:214]: 🤔 For some strange reason, even though we get the correct `fiscalYearStartMonth` here, that value isn't reflected when I render the dashboard using scenes.

https://github.com/grafana/grafana/assets/239999/f204a647-f693-4689-b92d-bd0a6eb1762d

Comment (torkelo) [reply public/app/features/dashboard-scene/serialization/transformSaveModelToScene.ts:214]: @axelavargas good catch! I only checked that it was reflected in the graph time ranges not in the time picker fiscal year selector itself 

Fixed here: 
https://github.com/grafana/scenes/pull/386

**Review by axelavargas — APPROVED** [review]

**Pull request merged**